README.md

```diff
@@ -9,6 +9,8 @@
 This extension provides capabilities for the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli/index.html) from directly within the VS Code editor. Use the VS Code command palette to quickly access all `ibmcloud dev` commands, without the need to leave the editor's context.
 
 ## Changelog
+- v1.0.0
+  - Removed cf commands
```

Comment:
Removing deprecated commands does not require moving to v1. I think this should be v0.3.0.

Reply:
Right. I think I marked the commands as deprecated in VSCode so what you said makes sense. I will fix it here.